src/components/_classes/component/Component.js

```diff
@@ -2391,6 +2391,18 @@ export default class Component extends Element {
       return false;
     }
 
+    if (flags.changed && !flags.changed.value) {
+      return false;
+    }
+
+    if (flags.changed && flags.changed.value && !row.values && flags.changed.instance.data !== row) {
```

Comment:
So, does this mean if I have a form and give it a label of "Values" (which will give it a key of "values"), that I should never calculate the value of it as long as the row context is different than the instance data?  I have to admit this is very strange, and does not seem to really solve the main problem.